lambdas/src/utils/health.ts

```diff
@@ -0,0 +1,58 @@
+import { EntityType } from 'dcl-catalyst-commons'
+import { SynchronizationState } from 'decentraland-katalyst-commons/synchronizationState'
+import { Logger } from 'log4js'
+import ms from 'ms'
+import { SmartContentClient } from './SmartContentClient'
+
+export enum HealthStatus {
+  HEALTHY = 'Healthy',
+  UNHEALTHY = 'Unhealthy',
+  DOWN = 'Down'
+}
+
+export async function refreshContentServerStatus(
+  contentService: SmartContentClient,
+  maxSynchronizationTime: string,
+  maxDeploymentObtentionTime: string,
+  logger: Logger
+): Promise<HealthStatus> {
+  let healthStatus: HealthStatus
+  try {
+    const fetchContentServerStatus = contentService.fetchContentStatus()
+    const [serverStatus, obtainDeploymentTime] = await Promise.all([
+      await fetchContentServerStatus,
+      await timeContentDeployments(contentService)
+    ])
+    const synchronizationDiffInSeconds = new Date(
```

Comment:
I think that `new Date(millis).getTime()` will just return the original `millis`